### DISCUSSION TITLE
.NET Core version from feed

### DISCUSSION THREAD

**Discussion by StanDeMan** [StanDeMan/RaspberryDebugger#6] [Ideas]
Maybe someone knows if there is a Microsoft feed (or service) for all .NET Core versions available?
Let me know, please. 
Thanks a lot!
Cheers!
StanDeMan

**Comment by Daniel127** [comment]
Hello,

Maybe this repo will help you, the JSON has links to other JSON that have all the information of each version.

https://github.com/dotnet/core/blob/main/release-notes/releases-index.json

https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json

Thanks for the work you have done, I hope to test a version with my new RaspBerry Pi Zero 2 64Bits.

**Reply by StanDeMan** [reply]
Hi Daniel127,

take the 64 Bit raspbian light version for the Pi Zero 2, please. I never run a .NET 64 Bit version on this tiny guy before. Let me know how it performs. I'm working on a html scraping mechanism to get all possible .NET 6 and 3.1LTS versions - .NET 5 is deprecated (not supported) since today- from the Microsoft download page. Integration is missing - the scraper is nearly done. Should be ready soon - hopefully. :) The main work was done by nforgeio and Jeff Lill - I basically keep the debugger alive. ;)

Working still on some stability issues and the download part. Try to release a new NuGet as soon as possible - combining the best for 64 and 32 Bit Raspbian versions. I'm doing my best...

Thanks and kindest regards
StanDeMan
